.github/workflows/publish-binaries.yml

```diff
@@ -0,0 +1,148 @@
+name: Releases prebuilt CLI binaries
+
+on:
+  workflow_call:
+    inputs:
+      goreleaser-config:
+        description: 'Config file for goreleaser; must match the goreleaser-config build.yml was called with'
+        required: true
+        type: string
+      goreleaser-flags:
+        description: 'Command-line flags to pass to goreleaser'
+        required: true
+        type: string
+      goreleaser-build-flags:
+        description: 'The goreleaser-flags arg that build.yml was called with'
+        required: true
+        type: string
+    secrets: {}
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  publish-binaries:
+    name: Publish Binaries
+    runs-on: macos-latest
+    steps:
+
+      # Section 0: checkout repo and install dependencies
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Fetch Tags
+        run: git fetch --quiet --prune --unshallow --tags
+      - name: Install pulumictl
+        uses: jaxxstorm/action-install-gh-release@v1.5.0
+        with:
+          repo: pulumi/pulumictl
+      - name: Install goreleaser-filter
+        uses: jaxxstorm/action-install-gh-release@v1.5.0
+        with:
+          repo: t0yv0/goreleaser-filter
+
+      # Section 1: configure
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
Will this be able to access secrets or do we need to thread through secrets: in the on: section I wonder.

Reply:
According to the docs `secrets: inherit` should do the trick.